.github/workflows/haskell.yml

```diff
@@ -8,12 +8,12 @@ jobs:
     runs-on: ubuntu-16.04
     strategy:
       matrix:
-        ghc: ["8.2.2", "8.4.4", "8.6.5", "8.8.1"]
+        ghc: ["8.2.2", "8.4.4", "8.6.5", "8.8.2"]
```

Comment:
I wanted this to be 8.8.3, but that’s [not supported yet](https://github.com/actions/setup-haskell/issues/8).